tests/publiccloud/run_ltp.pm

```diff
@@ -48,6 +47,47 @@ sub instance_log_args
         $self->{provider}->region);
 }
 
+sub upload_ltp_logs
+{
+    my ($self)        = @_;
+    my $log_file      = Mojo::File::path('ulogs/ltp_log.json');
+    my $ltp_testsuite = get_required_var('COMMAND_FILE');
+
+    upload_logs("$root_dir/ltp_log.raw",  log_name => 'ltp_log.raw',       failok => 1);
+    upload_logs("$root_dir/ltp_log.json", log_name => $log_file->basename, failok => 1);
+
+    return unless -e $log_file->to_string;
+
+    local @INC = ($ENV{OPENQA_LIBPATH} // testapi::OPENQA_LIBPATH, @INC);
+    eval {
+        require OpenQA::Parser::Format::LTP;
+
+        my $ltp_log = Mojo::JSON::decode_json($log_file->slurp());
+        my $parser  = OpenQA::Parser::Format::LTP->new()->load($log_file->to_string);
+
+        if (find_whitelist_testsuite($ltp_testsuite)) {
+            for my $result (@{$parser->results()}) {
+                next unless ($result->{result} ne 'ok');
+                my ($ltp_test) = grep { $_->{test_fqn} eq $result->{test_fqn} } @{$ltp_log->{results}};
+                for my $ret (@{$ltp_test->{test}->{retval}}) {
+                    if (override_known_failures($self, {%{$self->{ltp_env}}, result => $ret}, $ltp_testsuite, $result->{test_fqn})) {
```

Comment:
I think the condition should be reversed. Softfail only if **all** retvals are zero or match a whitelist entry.

Reply:
Makes sense, so if one of the retval's, do not match whitelist, we fail! THX!